.github/workflows/build.yml

```diff
@@ -12,7 +12,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Why is this removed

Reply:
This line did not actually solve the issue we were facing with #90 . Arguably it made it worst. It covered up rather than fixed compile errors. The actual problem we had with #90 was that it was not synced with master causing there to be compile errors. In retrospect, the compile errors in the build step was the correct behaviour to have.

Such a line may cause us to merge incompatible code to the master branch.